mk/config.mk

```diff
@@ -104,5 +104,12 @@ TA_SIGN_KEY ?= keys/default_ta.pem
 # may not because they obtain the isoc functions from elsewhere
 CFG_LIBUTILS_WITH_ISOC ?= y
 
-# Floating point support for TAs default on only for arm32 core
-CFG_TA_FLOAT_SUPPORT ?= $(CFG_ARM32_core)
+# Enables floating point support for user TAs
+# ARM32: EABI defines both a soft-float ABI and a hard-float ABI,
+#	 hard-float is basically a super set of soft-float. Hard-float
+#	 requires all the support routines provided for soft-float, but the
+#	 compiler may choose to optimize to not use some of them.
```

Comment:
"and use the floating-point registers instead"?
(sorry for my insistence, but that is what makes the link with CFG_WITH_VFP if I'm not mistaken ;))